testing/integration/testdata/.upgrade-test-agent-versions.yml

```diff
@@ -5,8 +5,7 @@
 # upgrade integration tests.
 
 testVersions:
+  - 9.0.4-SNAPSHOT
   - 9.0.3
-  - 9.0.3-SNAPSHOT
-  - 8.19.0-SNAPSHOT
```

Comment:
`8.19` is an LTS branch so this shouldn't be removed, I plan to open a PR to address that so hold your horses

Reply:
Maybe we can still merge this one in the meantime and it will get fixed by your PR then?

Reply:
ok this is now covered by this one https://github.com/elastic/elastic-agent/pull/8783, so when I merge it I will make sure to close this one 🙂